components/panelbar/expanded-items.md

```diff
@@ -207,6 +209,100 @@ If the [ExpandMode](#expandmode) is set to `Single` the ExpandedItems collection
 }
 ````
 
+## Programmatically Expand and Collapse Items
+
+>caption Programmatically expand and collapse items on button click
+
+````CSHTML
+@* Use the buttons to programmatically expand and collapse items in the PanelBar *@ 
+
+<TelerikButton OnClick="@ExpandItems">Expand First and Second Item</TelerikButton>
+<TelerikButton OnClick="@CollapseItems">Collapse the Expanded Item</TelerikButton>
+
+<div style="width: 30%;">
+    <TelerikPanelBar Data="@Items"
+                     @bind-ExpandedItems="@ExpandedItems">
+    </TelerikPanelBar>
+</div>
+
+@code {
+    public List<PanelBarItem> Items { get; set; }
+    public IEnumerable<object> ExpandedItems { get; set; } = new List<object>();
```

Comment:
How important it is to use the `PanelBarItem` type vs `object` in the IEnumerable and List?